assembl/static2/js/app/components/common/youtubeTheater.jsx

```diff
@@ -0,0 +1,52 @@
+// @flow
+import React from 'react';
+import get from 'lodash/get';
+
+export default class YoutubeTheater extends React.Component {
+  state: {
+    theaterMode?: boolean
+  };
+
+  timeout: number;
+
+  closeTheater = () => this.setState({ theaterMode: false });
+
+  openTheater = () => this.setState({ theaterMode: true });
+
+  render = () => {
+    const { videoId } = this.props;
+    const theaterMode = get(this, 'state.theaterMode', false);
```

Comment:
I'd prefer you initialize `this.state = { theaterMode: false }` in constructor instead of doing that.